main.ts

```diff
@@ -1,15 +1,27 @@
-import { App, Plugin, PluginSettingTab, Setting, TAbstractFile } from 'obsidian';
-// @ts-ignore
+import { App, Plugin, PluginSettingTab, Setting, TAbstractFile, FuzzySuggestModal, FuzzyMatch, Notice } from 'obsidian';
+import * as obsidian from 'obsidian';
 import compareVersions from 'compare-versions';
 
 interface CustomJSSettings {
   jsFiles: string;
   jsFolder: string;
+  startupScriptNames: string;

```

Comment:
You may want to make this an array like the registered scripts so the user can select and add them like the registered scripts in settings. Makes it a bit more consistent.